runtime/battery-station/src/parameters.rs

```diff
@@ -74,6 +75,26 @@ parameter_types! {
     /// the juror has to pay.
     pub const StakeWeight: u128 = 2 * BASE;
 
+    // Global Disputes
+    /// Pallet identifier
+    pub const GlobalDisputesPalletId: PalletId = GLOBAL_DISPUTES_PALLET_ID;
+    /// Vote lock identifier, mainly used for the LockableCurrency on the native token.
+    pub const VoteLockIdentifier: LockIdentifier = *b"zge/vote";
+    /// The maximum number of owners for a voting outcome for private API calls of `push_voting_outcome`.
+    pub const MaxOwners: u32 = 10;
+    /// The maximum number of market ids (participate in multiple different global disputes at the same time) for one account to vote on outcomes.
+    pub const MaxGlobalDisputeVotes: u32 = 50;
+    /// The minimum required amount to vote on an outcome.
+    pub const MinOutcomeVoteAmount: Balance = 10 * CENT;
+    /// The maximum number of voting outcomes.
+    pub const MaxOutcomeLimit: u32 = u32::MAX;
+    /// The minimum number of voting outcomes.
+    pub const MinOutcomes: u32 = 2;
```

Comment:
Good catch again! I don't really understand the question. `MinOutcomes` is an old constant, which is not used anymore, so it can get deleted. `zrml-prediction-market` completely controls the start of a global dispute now. 

beaccb9aff84d1b048cc2e490545ead85aaa20c8